code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -70,7 +70,7 @@
 
 /obj/item/gun/magic/staff/healing
 	name = "staff of healing"
-	desc = "An artefact that spits bolts of restoring magic which can remove ailments of all kinds and even raise the dead."
+	desc = "An artefact that spits bolts of restoring magic which can remove ailments of all kinds and even raise the dead. On the handle is a beautiful engraving in High Spaceman, \"Thou shalt not crosseth thy beams.\""
```

Comment:
this should be attached in examine instead, when the user is not `is_wizard_or_friend` because normal wizards just shoot  bolts

Reply:
but thats not how engravings work

Reply:
It's a magic staff, why would wizards care how engravings are supposed to work.

Reply:
you right